scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -12,6 +12,18 @@ <h5 class="modal-title" id="causativeLabel">Mark variant as causative</h5>
         <form action="{{ url_for('cases.mark_causative', institute_id=institute._id, case_name=case.display_name,
            variant_id=variant._id, partial_causative=False) }}" method="POST">
           <div class="modal-body">
+            <div class="row">
```

Comment:
This looks like a bit of an overkill to me? Isn't it enough to be able to tag a case from the case page? I would keep the functionality as simple as possible

Reply:
This was one of the wishes from the users - it was felt this should appear especially on the dialog for marking causative to ensure it is used and doesn't feel like an extra step to take. The reason it feels like overkill is probably that I merged three kinds of tags into one system to save some real estate. The provisional/definitive diagnosis, request for techincal/medical evaluation and categories of diagnostic findings that we cannot yet (fully) pin/mark.